lib/situation_room/repo.ex

```diff
@@ -0,0 +1,5 @@
+defmodule Situation_Room.Repo do
```

Comment:
I didnt notice this in the PR yesterday, but can we rename this module `SituationRoom`  (upper camel case vs. snake case) and remove the underscore. Both here any any other places in the app this namespace is mentioned.

The directory naming is correct, just update the Elixir module naming.

Reply:
'Situation_Room' references in the Application and Repo have been renamed to 'SituationRoom'